pconnector/client/option.go

```diff
@@ -76,6 +77,17 @@ func WithDelve(port int) Option {
 	})
 }
 
+func WithEnvVar(key, value string) Option {
+	return serveConfigFunc(func(in *plugin.ClientConfig) error {
+		if in.Cmd == nil {
+			return errors.New("plugin client config has no Cmd set (tip: check if WithReattachConfig is used)")
```

Comment:
Hmm, will this affect the tests in Conduit? I haven't reviewed the Conduit PR yet, maybe the answer lies there.

Reply:
Currently, `WithEnvVar` isn't used in tests in Conduit. My reasoning was that `in.Cmd` should not be nil in normal circumstances, and `WithReattachConfig` (that sets `Cmd` to `nil`) is used in tests only.